Assets/MRTK/Core/Inspectors/PropertyDrawers/SceneInfoUtils.cs

```diff
@@ -158,7 +158,7 @@ public void OnProcessScene(Scene scene, BuildReport report)
                     changed = true;
                 }
 
-                
+
```

Comment:
nit: this removes the extra whitespace, but there's also a duplicate blank line here